.github/workflows/deploy-examples.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy example apps to Fly
+
+on:
+  # Allow calling this workflow from other workflows
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+    secrets:
+      FLY_API_TOKEN:
+        description: "Fly API token for deploying apps"
+        required: true
+
+  # Allow calling this workflow manually from the GitHub UI
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Wasp version to use for deployment (default to latest)"
```

Comment:
The picture mentions GHC and Node versions, but this mentions the Wasp version.

How come?

Also, should we figure out the Wasp version from the `version` field in the app, have a single source of truth?

Reply:
> The picture mentions GHC and Node versions

The picture is from another workflow. As I mentioned, I am not able yet to get the workflow dispatch trigger to work until the workflow hits `main`

---

> Also, should we figure out the Wasp version from the version field in the app, have a single source of truth?

I don't think so. If we want to e.g. release these projects with a pre-release version, we need a way of saying so. Otherwise, these examples that live in the wasp repo are using the repo's version, so, latest.

Reply:
My reasoning is this is using the released version of wasp, so it should also use the release version of the apps, not the one in `main`.

Reply:
But it will still end up on the main branch at some point. A week from now, this action will be both in `main` and on `release`, so why not start from main and follow the normal flow immediately?

Reply:
Okay. I will keep it on `release` for now while doing tests, but change to `main` before merging.